spec/javascripts/reducers/project_spec.js

```diff
@@ -0,0 +1,36 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/project";
+
+describe("Project Reducer",() => {
+  describe("RECEIVE_PROJECT",() => {
+    it("returns data from action",() => {
```

Comment:
This one sounds kinda confusing to me, can we improve the description so it tells what exactly this action does?